### ISSUE TITLE
The following text hangs the conversion

### ISSUE THREAD

**Issue by solariq** [mdiep/MMMarkdown#100]
The text is in russian, but the problem seems to be in "\n\t" combination. 

```swift
let text = "\nОбновление системы, по условиям российского диллера, производится только в сервисном центре. Самостоятельные манипуляции с версией прошивки проводятся на свой страх и риск.\n\nДля обновления прошивки вам понадобится файл прошивки и usb флеш-накопитель. Далее произвести следующий порядок действий\n\n* флешку отформатировать в FAT систему\n* сохранить все личные данные на MicroSD устройства на внешнем носителе\n* записать файл прошивки в корень файловой системы\n* подключить флешку к головному устройству\n\t* пройти в меню Настройка -&gt; Система -&gt; Обновление системы -&gt; Ок\n\t\n\tили\n\n\t*  пройти в  меню Настройка -&gt; Система -&gt; Сброс -&gt; подтвердить\n\n\tили\n\n\t*  перезагрузить устройство \n\n* (если появится меню) ввести предложенный пин-код и нажать auto update\n* когда появится диалоговое окно на китайском языке - нажать на левую кнопку\n* выбрать язык и/или цвет и/или устанавливаемые приложения\n* при запросе выбора системы интерфейса выбрать \"FlyAudioUI\" и нажать \"всегда\"\n"
try! MMMarkdown.HTMLStringWithMarkdown(text, extensions: [
                    .AutolinkedURLs,
                    .FencedCodeBlocks,
                    .Tables,
                    .UnderscoresInWords,
                    .Strikethroughs,
                    .GitHubFlavored
                    ])
```

**Comment by mdiep** [comment]
Thanks for the report! I've fixed the issue and created a new release, 0.5.3.

**Comment by solariq** [comment]
Matt, thanks a lot for your quick response and fix!

IK
